armada_command/command_run.py

```diff
@@ -185,18 +234,15 @@ def command_run(args):
         run_command += ' --hidden_is_restart'
     payload['run_command'] = run_command
 
-    # ---
-    if verbose:
-        print('payload: {0}'.format(payload))
 
-    warn_if_hit_crontab_environment_variable_length(payload['environment'])
-
-    print('Checking if there is new image version. May take few minutes if download is needed...')
-    result = armada_api.post('run', payload, ship_name=ship)
+def _print_result(result, is_restart):
+    if not result:
+        print("ERROR: armada API call failed.")
+        sys.exit(1)
```

Comment:
We should raise exception instead and have one place to handle errors, which is: https://github.com/armadaplatform/armada/blob/766ea71c6cc65d291c95f5a48807c414842718c6/armada_command/armada.py#L154